examples/opencv_rc.py

```diff
@@ -0,0 +1,541 @@
+                    linear = Direction.NONE
+
+                # Stop moving backward
+                if char == 's' and self._action['linear'] == Direction.BACKWARD:
+                    linear = Direction.NONE
+
+                # Stop turning left
+                if char == 'a' and self._action['angular'] == Direction.LEFT:
+                    angular = Direction.NONE
+
+                # Stop turning right
+                if char == 'd' and self._action['angular'] == Direction.RIGHT:
+                    angular = Direction.NONE
+
+                # Set the new actions
+                self._set_action(linear, angular)
+
+    @property
+    def head_light(self):
+        return self._head_light
+
+    @head_light.setter
+    def head_light(self, value):
+        """
+        Toggle the state of the light.
+        :param value: Bool. True for turning the light on, False otherwise.
+        :return: None
+        """
+
+        if self._head_light != value:
+            # Set the value
+            self._head_light = value
+
+            # Tell Cozmo to actually turn the light on/off
+            self._cozmo_clt.set_head_light(enable=self._head_light)
+
+    @property
+    def head_tilt(self):
+        return self._head_tilt
+
+    @head_tilt.setter
+    def head_tilt(self, value):
+        """
+        Set the head tilt and tell Cozmo to actually move its head.
+        :param value: Float. The new head angle in radians.
+        :return: None
+        """
+
+        # Set the value of the head tilt
+        self._head_tilt = max(pc.MIN_HEAD_ANGLE.radians,
+                              min(pc.MAX_HEAD_ANGLE.radians, value))
+
+        # Send the command to cozmo
+        self._cozmo_clt.set_head_angle(self._head_tilt)
+
+    @property
+    def lift_height(self):
+        return self._lift_height
+
+    @lift_height.setter
+    def lift_height(self, value):
+        """
+        Set the lift height and tell Cozmo to actually move the lift.
+        :param value: Float. The new horizontal position of the lift in mm.
+        :return: None
+        """
+
+        # Set the value of the lift height
+        self._lift_height = max(pc.MIN_LIFT_HEIGHT.mm,
+                                min(pc.MAX_LIFT_HEIGHT.mm, value))
+
+        # Send the command to cozmo
+        self._cozmo_clt.set_lift_height(height=self._lift_height)
+
+    @property
+    def linear_velocity(self):
+        return self._velocity['linear']
+
+    @linear_velocity.setter
+    def linear_velocity(self, value):
+        self._velocity['linear'] = max(0, min(value, pc.MAX_WHEEL_SPEED.mmps))
+
+    @property
+    def angular_velocity(self):
+        return self._velocity['angular']
+
+    @angular_velocity.setter
+    def angular_velocity(self, value):
+        self._velocity['angular'] = max(0,
+                                        min(value, pc.MAX_WHEEL_SPEED.mmps /
+                                            pc.TRACK_WIDTH.mm))
```

Comment:
Just so you know, we are using 120 characters per line in pycozmo.
I don't know if it's very important but I personally like better longer lines ;)

If you want, you can check the [flake8 configuration](https://github.com/zayfod/pycozmo/blob/master/.flake8)

Reply:
Thank you for pointing that out, I was not aware of it. I thought I was stuck with 80 characters per line.
I 100% agree with you on longer lines. Especially, on more modern screens where there is so much space. I will configure my IDE to put the cut of at 120 characters from now on (that is going to be liberating).